### PR TITLE
added top menu 

### PR DESCRIPTION
**Pull request by ckiss**

### CODE REVIEW

File: client/components/sidebar.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ export default React.createClass({
         </a>
```
Comment (mcdonnelldean) [inline client/components/sidebar.js]: Should be removed

File: client/components/header.js [hunk 2]
```diff
@@ -7,21 +7,29 @@ export default React.createClass({
...
-            </a>
-          </li>
-          <li>
-            <Link to={'/logout'} className="has-icon has-icon-signout">
-              <span className="icon icon-signout"></span>
-              <span>Sign out</span>
-            </Link>
-          </li>
-        </ul>
+      <div className="col-xs-4 col-sm-6 nav-main-wrapper">
```
Comment (mcdonnelldean) [inline client/components/header.js]: Break out menu component like vidi

**Comment by mcdonnelldean** [comment]
@ckiss You should be able to remove all the sidebar stuff now

**Comment by ckiss** [comment]
@mcdonnelldean why don't we keep both and display only one of them depending on the device? it seems to mobile oriented on desktop with only that menu at the top. we can use this or something similar: https://www.npmjs.com/package/mobile-detect

**Comment by mcdonnelldean** [comment]
Because it breaks layout further down,

I'd rather we focused on functionality that worrying about menu orientation.

Kindest Regards,

Dean

**Comment by mcdonnelldean** [comment]
@ckiss Can you get this finished out? I need to make sure both layouts are on par otherwise it becomes impossible for Karolina to manage. We can revisit sidebar you another time. Right now a clean mobile layout is priority since it's such a strong selling point. There are more changes to come on this.

**Comment by ckiss** [comment]
@mcdonnelldean sidebar removed now

**Pull request merged**